markdown/events/2022/bytesize-36-multiqc.md

```diff
@@ -24,3 +24,119 @@ It is our hope that these talks / videos will build an archive of training mater
 
 This week, Phil Ewels ([@ewels](https://github.com/ewels/)) will talk about MultiQC (of which he is the author).
 He'll cover customising reports to have consistent branding, additional rich content and more.
+
+<details markdown="1"><summary>Video transcription</summary>
+**Note: The content has been edited for reader-friendliness**
+
+[0:01](https://www.youtube.com/watch?v=a3_IYSMrKAk&t=1)
+(host) Hello and welcome everybody to another talk of the bytesize talk series that is offered by the nf-core community. We should mention that it is receiving support from the Chan Zuckerberg Initiative. We're thankful for that. Today Phil Ewels is back and he will tell us more about MultiQC and how to customize MultiQC reports for example, for your own pipeline. Thanks for joining us, Phil, today.
+
+(speaker) Thank you for having me.
+
+[0:33](https://www.youtube.com/watch?v=a3_IYSMrKAk&t=33)
+Sorry that I've ended up doing two bytesize talks in two weeks. It's been a bit of a reschedule shuffle. Hopefully you won't be too tired of my voice already. Today's talk is a bit of a break from what I've spoken about previously with bytesize, in that it doesn't really talk about nf-core at all. This talk is purely about MultiQC which is one of my other pet projects which I've been working on for a few years now. But MultiQC is used very heavily within the majority of nf-core pipelines. We figure it's a relevant topic for most nf-core developers certainly, but also people using nf-core pipelines as well. Today I'm gonna start off with a quick introduction just for those people who might be watching who have no idea what MultiQC is. Then I'll talk about a few tips for people developing pipelines for recommendations to get the most out of MultiQC. A few recommendations for people who are running nf-core pipelines and this is most relevant for people working in facilities or large scale routine processing places. But of course it can be used by anyone.
```

Comment:
```suggestion
Sorry that I've ended up doing two bytesize talks in two weeks. It's been a bit of a reschedule shuffle. Hopefully you won't be too tired of my voice already. Today's talk is a bit of a break from what I've spoken about previously with bytesize, in that it doesn't really talk about nf-core at all. This talk is purely about MultiQC which is one of my other pet projects which I've been working on for a few years now. But MultiQC is used very heavily within the majority of nf-core pipelines. We figure it's a relevant topic for most nf-core developers certainly, but also people using nf-core pipelines as well. Today I'm gonna start off with a quick introduction just for those people who might be watching who have no idea what MultiQC is. Then I'll talk about a few tips for people developing pipelines and recommendations how to get the most out of MultiQC. A few recommendations for people who are running nf-core pipelines and this is most relevant for people working in facilities or large scale routine processing places. But of course it can be used by anyone.
```

Reply:
minor change in third to last sentence of this paragraph to improve language

> "Then I'll talk about a few tips for people developing pipelines ~~for~~ **and** recommendations **how** to get the most out of MultiQC."